tests/test_cli/test_cli_terminal_tool_integration.py

```
@@ -40,12 +40,40 @@ def setUp(self):
+            'test_python_code_chars.txt'
+        ]
+
+        for pattern in leaked_files:
+            if '*' in pattern:
+                # Handle glob patterns
+                import glob
+                for file in glob.glob(os.path.join(self.original_cwd, pattern)):
+                    try:
+                        os.remove(file)
+                    except Exception:
+                        pass
```

Ruff (0.13.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)